### PR TITLE
naming: use 'colormap' in favor of 'cmap'

### LINKED ISSUE #96: cmap vs colormap on a layer
## ❓ Questions and Help

As initially posed in #89 - both `cmap` and `colormap` exist as properties on a layer, but have slightly different setters. It's not clear to me what the difference is or why we need both. If possible it would be nice to eliminate one. Maybe @kne42 has the answer?

### PR DESCRIPTION
**Pull request by kne42**
closes #96

### CODE REVIEW

File: napari/layers/_image_layer/model.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def __init__(self, image, meta=None, multichannel=None, *, name=None, **kwargs):
         self._image = image
         self._meta = meta
         self.cmap = Image.default_cmap
-        self.interpolation = Image.default_interpolation
+        self.interpolation = self.default_interpolation
```
Comment (jni) [inline napari/layers/_image_layer/model.py:101]: Why change this line but not the line above it?

**Review by jni — APPROVED** [review]
I'm not positive that this is the best choice (rather than the reverse), but happy to revisit later, and I agree that it should be a single variable, no aliases. (In the docs and docstrings, we should specify that this is the same as colormap *and* LUT in some circles.)

**Comment by jni** [comment]
Somehow, this already needs a rebase/master merge. 😂

**Comment by sofroniewn** [comment]
I'm open to `colormap` instead of `cmap` - maybe my fault that we went with the shorter one, but now that I think about it more, maybe it is more clear to use `colormap`. Sorry @kne42!

**Comment by kne42** [comment]
I've reversed it so that `cmap` is replaced by `colormap`. If everyone's ok with this we can go ahead and merge :)

**Comment by jni** [comment]
Boom. Thanks for the fast response @kne42!

**Pull request merged**